index.bs

```diff
@@ -4183,9 +4183,9 @@ The [=remote end event trigger=] is:
 
 <div algorithm="remote end event trigger for browsingContext.contexDestroyed">
 
-Define the following [=browsing context tree discarded=] steps:
-
-1. Let |context| be the browsing context being discarded.
+The [=remote end event trigger=] is
+the <dfn export>WebDriver BiDi navigable destroyed</dfn> steps given
+<var ignore>navigable</var> and |context|:
 
 1. Let |params| be the result of [=get the browsing context info=], given
```

Comment:
`|context|` now seems to be undefined here.

Reply:
I think it is defined on line 4188 or do you mean something else?

Reply:
Oh, sorry, I misread it.